data-plane/receiver/src/main/java/dev/knative/eventingkafkabroker/receiver/RequestToRecordMapper.java

```diff
@@ -0,0 +1,22 @@
+package dev.knative.eventingkafkabroker.receiver;
+
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.kafka.client.producer.KafkaProducerRecord;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+
+public interface RequestToRecordMapper<K, V> extends
```

Comment:
Can you convert this to future based apis?

`extends Function<HttpServerRequest, Future<KafkaProducerRecord<K, V>>>`

Reply:
Done